src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
-body {
+@import "./header.scss";
+@import "./card.scss";
+@import "./stars.scss";
+@import "./queries.scss";
+
+* {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
   margin: 0;
+  --main-color: #00acdc;
+  --text-color: #616070;
+  --value-color: #060b35;
```

Comment:
Use SCSS variables instead of default CSS variables
https://sass-lang.com/documentation/variables